dotnet/build-packages-dev.sh

```diff
@@ -1,21 +1,27 @@
-DEVELOPMENT_VERSION="0.2.114-prerelease"
+DEVELOPMENT_VERSION="$1"
+TRACER_SHA="$2"
+
 AGENT_DOWNLOAD_URL="http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.35.2-1-x86_64.zip"
-TRACER_DOWNLOAD_URL="https://apmdotnetci.blob.core.windows.net/apm-dotnet-ci-artifacts-master/INSTALL_SHA/windows-tracer-home.zip"
+TRACER_DOWNLOAD_URL="https://apmdotnetci.blob.core.windows.net/apm-dotnet-ci-artifacts-master/$TRACER_SHA/windows-tracer-home.zip"
+
+echo "Downloading tracer from $TRACER_DOWNLOAD_URL"
+#wget -O tracer.zip $TRACER_DOWNLOAD_URL
```

Comment:
Does this need to be uncommented?

Reply:
done in 802a5e5bd8c9aeedee6bf27ef58afa921a09e7e5